lifecycle-operator/main.go

```diff
@@ -275,16 +275,8 @@ func main() {
 
 	workloadVersionLogger := ctrl.Log.WithName("KeptnWorkloadVersion Controller").V(determineWorkloadVersionControllerLogLevel(env.KeptnWorkloadVersionControllerLogLevel, env.KeptnWorkloadInstanceControllerLogLevel))
 	workloadVersionRecorder := mgr.GetEventRecorderFor("keptnworkloadversion-controller")
-	workloadVersionReconciler := &keptnworkloadversion.KeptnWorkloadVersionReconciler{
-		SchedulingGatesHandler: controllercommon.NewSchedulingGatesHandler(mgr.GetClient(), workloadVersionLogger, env.SchedulingGatesEnabled),
-		Client:                 mgr.GetClient(),
-		Scheme:                 mgr.GetScheme(),
-		Log:                    workloadVersionLogger,
-		EventSender:            controllercommon.NewEventMultiplexer(workloadVersionLogger, workloadVersionRecorder, ceClient),
-		Meters:                 keptnMeters,
-		TracerFactory:          telemetry.GetOtelInstance(),
-		SpanHandler:            spanHandler,
-	}
+	workloadVersionReconciler := keptnworkloadversion.NewReconciler(mgr.GetClient(), mgr.GetScheme(), controllercommon.NewEventMultiplexer(workloadVersionLogger, workloadVersionRecorder, ceClient),
```

Comment:
please split this line up so that each parameter passed to the NewReconciler function is in a separate line, otherwise this is quite hard to read

Reply:
Alright, let me do that now